meson-mode.el

```diff
@@ -925,7 +925,7 @@ Return either `line-beginning-position' of the matching line or nil."
 (defun meson-lookup-doc (identifier)
   "Open Meson reference manual and find the function or object named IDENTIFIER.
 Return the buffer containing the reference manual or nil."
-  (interactive (list (thing-at-point 'symbol)))
+  (interactive (list (or (meson-function-at-point) (thing-at-point 'symbol))))
```

Comment:
I'm not sure about the order here. If I'm inside:
```meson
executable(..., cpp_args : cxx.get_supported_arguments(['-Wsuggest-attribute=const']))
```
then pressing F1 at get_supported_arguments gives me documentation for `executable`, which is probably not, what I wanted. Maybe, with the opposite order, "fails" in some situations as well. What do you think would be better?

Reply:
I have initially chosen this order, partly because `thing-at-point` would match *any* symbol at all and partly because of me misunderstanding your second example. After some experimenting, I now agree that the reverse order is better, as it’s behavior is less surprising to someone used to `thing-at-point` based commands, it enables more granular input based on `point` and it doesn’t need to compute `meson-function-at-point` on every invocation.

Reply:
Changed in 9415e589066bbb62f610353f4daa07b6bc60646b.